experiment/build/gcb_build.py

```diff
@@ -39,84 +40,76 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer=None, benchmark=None):
+    return docker_images.get_images_to_build([fuzzer], [benchmark])
+
+
 def build_base_images():
     """Build base images on GCB."""
-    _build(get_build_config_file('base-images.yaml'), 'base-images')
+    image_templates = {'base-image': _get_buildable_images()['base-image']}
+    config = generate_cloudbuild.create_cloud_build_spec(image_templates,
+                                                         build_base_images=True)
+    _build(config, 'base-images')
 
 
 def build_coverage(benchmark):
     """Build coverage image for benchmark on GCB."""
-    coverage_binaries_dir = exp_path.filestore(
-        build_utils.get_coverage_binaries_dir())
-    substitutions = {
-        '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
-        '_BENCHMARK': benchmark,
+    buildable_images = _get_buildable_images(benchmark=benchmark)
```

Comment:
You can't pass `fuzzer=coverage` to this function?

Reply:
It can be passed, but it would be like a dummy variable because image_types.yaml has separate coverage images. Should I add that here ?

Reply:
I didn't want for a dummy variable. I meant if you passed coverage would it return the coverage images. I think the answer is no so don't pass it.